src/_imagingft.c

```diff
@@ -790,7 +791,13 @@ font_render(FontObject* self, PyObject* args)
     int index, error, ascender, horizontal_dir;
     int load_flags;
     unsigned char *source;
-    FT_GlyphSlot glyph;
+    FT_Glyph glyph;
+    FT_GlyphSlot glyphSlot;
```

Comment:
Be consistent with the other variables, use `snake_case`:
```suggestion
    FT_GlyphSlot glyph_slot;
```